### PR TITLE
Add info about checking version + updating

### PR DESCRIPTION
**Pull request by kingscott**
In regards to #5484. I added a section to `/docs/installation/` to show how to check your version of Jekyll, find the latest version and update it. 

Open to feedback.

### CODE REVIEW

File: site/_docs/installation.md [hunk 1]
```diff
@@ -103,4 +103,25 @@ Check out [the extras page](../extras/) for more information.
...
 </div>
 
-Now that you’ve got everything installed, let’s get to work!
+## Already Have Jekyll?
+
+Before you start developing with Jekyll, you may want to check that you're up to date with the latest version. To find your version of Jekyll, run one of these commands:
+
+```sh
+$ jekyll --version
+$ gem list jekyll
```
Comment (pathawks) [inline site/_docs/installation.md]: If we're trying to encourage the use of Bundler everywhere, maybe we should change this to something like

```
bundle exec jekyll --version
bundle show jekyll
```

Maybe that requires a larger refactoring of this document than is in the scope of this PR

Comment (kingscott) [inline site/_docs/installation.md]: That could be done! Was just trying to keep it scoped; open for change though 👍

Comment (pathawks) [inline site/_docs/installation.md]: Probably best to take care of that in a separate PR :+1:

File: site/_docs/installation.md [hunk 2]
```diff
@@ -103,4 +103,25 @@ Check out [the extras page](../extras/) for more information.
...
+```
+
+You can also use <a href="https://rubygems.org/gems/jekyll">RubyGems</a> to find the current versioning of any gem. But you can also use the `gem` command line tool:
```
Comment (pathawks) [inline site/_docs/installation.md]: Since this is Markdown, let's use Markdown style links:

```markdown
[RubyGems](https://rubygems.org/gems/jekyll)
```

**Comment by pathawks** [comment]
Fixes #5484

**Comment by ashmaroli** [comment]
nice write-up @kingscott, may I ask you to include a line about `gem outdated ` as well, for completion sakes? Thanks.

**Comment by kingscott** [comment]
Sure, how's that?

**Comment by kingscott** [comment]
Awesome, thanks1

**Comment by kingscott** [comment]
Any idea about the errors, not sure why the build fails?

**Comment by ashmaroli** [comment]
Dont worry about the failing tests.. its a known bug.. totally unrelated to this PR.

**Comment by DirtyF** [comment]
This only covers one way to update Jekyll, we can already publish this and complete with bundler method later.

**Comment by pathawks** [comment]
:shipit:

**Comment by parkr** [comment]
@jekyllbot: merge +site

**Comment by kingscott** [comment]
Sweet, thanks guys!

**Comment by parkr** [comment]
@kingscott Thank *you*! 😄

**Pull request merged**